FhirToDataLake/global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sdk": {
+      "version": "5.0.403",
+      "rollForward": "latestFeature"
```

Comment:
Does this mean if the specific version `x.y.z` is not found, we will roll the version forward to the latest `x.y.z'`?